cli/utils/api.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/kubeshop/tracetest/cli/analytics"
+	"github.com/kubeshop/tracetest/cli/config"
+	"github.com/kubeshop/tracetest/cli/openapi"
+)
+
+func GetAPIClient(cliConfig config.Config) *openapi.APIClient {
+	config := openapi.NewConfiguration()
+	config.AddDefaultHeader("x-client-id", analytics.ClientID())
+	config.Scheme = cliConfig.Scheme
+	config.Host = strings.TrimSuffix(cliConfig.Endpoint, "/")
+	if cliConfig.ServerPath != nil {
+		config.Servers = []openapi.ServerConfiguration{
+			{
+				URL: *cliConfig.ServerPath,
+			},
+		}
+	}
+
+	return openapi.NewAPIClient(config)
+}
+
+func GetHttpClient(cliConfig config.Config) http.Client {
+	transport := &http.Transport{
+		DialContext: (&net.Dialer{
+			Timeout: time.Second,
+		}).DialContext,
+	}
+
+	client := http.Client{
+		Transport: transport,
+		Timeout:   30 * time.Second,
+	}
```

Comment:
Both timeouts could be constants to make them explicit and to allow us to refactor and expose them as a CLI configuration of the future.